browser/src/react-redux/App/ForgotPasswordPage/ForgotPasswordPage.react.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const initialState = {
+  email: '',
+  response: '',
+};
+
+class ForgotPasswordPage extends Component {
+  state = initialState;
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    this.setState({ response: '' });
+    const { email } = this.state;
+    const userEmail = {
+      email,
+    };
+    axios
+      .post('/api/auth/forgot', userEmail)
+      .then(() => {
+        this.setState({ response: 'You should receive an email, soon, with instructions to reset your password.' });
+      })
+      .catch((error) => {
+        const { message } = error.response.data;
+        this.setState({ response: message });
```

Comment:
Let's create a new property on state called `errorMessage`. We can update it with this message. Right now, there's no distinguishing characteristics about the error vs success messages. Once we style this out though, we'll want to know whether it was successful or not.